Documentation/contributing/testing/e2e_legacy.rst

```diff
@@ -0,0 +1,739 @@
+    Running Suite: runtime
+    ======================
+    Random Seed: 1516125117
+    Will run 42 of 164 specs
+    ................
+    RuntimePolicyEnforcement Policy Enforcement Always
+      Always to Never with policy
+      /Users/ianvernon/go/src/github.com/cilium/cilium/test/runtime/Policies.go:258
+    •
+    ------------------------------
+    RuntimePolicyEnforcement Policy Enforcement Always
+      Always to Never without policy
+      /Users/ianvernon/go/src/github.com/cilium/cilium/test/runtime/Policies.go:293
+    •
+    ------------------------------
+    RuntimePolicyEnforcement Policy Enforcement Never
+      Container creation
+      /Users/ianvernon/go/src/github.com/cilium/cilium/test/runtime/Policies.go:332
+    •
+    ------------------------------
+    RuntimePolicyEnforcement Policy Enforcement Never
+      Never to default with policy
+      /Users/ianvernon/go/src/github.com/cilium/cilium/test/runtime/Policies.go:349
+    .................
+    Ran 42 of 164 Specs in 0.002 seconds
+    SUCCESS! -- 0 Passed | 0 Failed | 0 Pending | 122 Skipped PASS
+
+    Ginkgo ran 1 suite in 1.830262168s
+    Test Suite Passed
+
+The output has been truncated. For more information about this functionality,
+consult the aforementioned Ginkgo documentation.
+
+.. _running_k8s_tests:
+
+Running Kubernetes Tests
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+To run all of the Kubernetes tests, run the following command from the ``test`` directory:
+
+.. code-block:: shell-session
+
+    INTEGRATION_TESTS=true ginkgo --focus="K8s"
+
+To run a specific test from the Kubernetes tests suite, run the following command
+from the ``test`` directory:
+
+.. code-block:: shell-session
+
+    INTEGRATION_TESTS=true ginkgo --focus="K8s.*Check iptables masquerading with random-fully"
+
+Similar to the Runtime test suite, Ginkgo searches for all tests in all
+subdirectories that are "named" beginning with the string "K8s" and
+contain any characters after it.
+
+The Kubernetes tests support the following Kubernetes versions:
+
+* 1.16
+* 1.17
+* 1.18
+* 1.19
+* 1.20
+* 1.21
+* 1.22
+* 1.23
+* 1.24
+* 1.25
+* 1.26
+
+By default, the Vagrant VMs are provisioned with Kubernetes 1.23. To run with any other
+supported version of Kubernetes, run the test suite with the following format:
+
+.. code-block:: shell-session
+
+    INTEGRATION_TESTS=true K8S_VERSION=<version> ginkgo --focus="K8s"
+
+.. note::
+
+   When provisioning VMs with the net-next kernel (``NETNEXT=1``) on
+   VirtualBox which version does not match a version of the VM image
+   VirtualBox Guest Additions, Vagrant will install a new version of
+   the Additions with ``mount.vboxsf``. The latter is not compatible with
+   ``vboxsf.ko`` shipped within the VM image, and thus syncing of shared
+   folders will not work.
+
+   To avoid this, one can prevent Vagrant from installing the Additions by
+   putting the following into ``$HOME/.vagrant.d/Vagrantfile``:
```

Comment:
Use clear, plain language
```suggestion
   To avoid this, prevent Vagrant from installing the Additions by
   putting the following into ``$HOME/.vagrant.d/Vagrantfile``:
```